app/controllers/player_controller.rb

```diff
@@ -0,0 +1,22 @@
+class PlayerController < ApplicationController
```

Comment:
__[reek]__
* [InstanceVariableAssumption] assumes too much for instance variable @slide [:ref](https://github.com/troessner/reek/blob/master/docs/Instance-Variable-Assumption.md)

<!-- Comment by [SideCI](https://sideci.com) -->